docs/digitalocean.rst

```diff
@@ -0,0 +1,178 @@
+
+This means manage.py will be in /home/user/ProjectName
+
+Install Nginx & uWSGI
+----------------
+
+Next we need to install Nginx, which will make setting up SSL easy and uWSGI which is how Django and Nginx talk.
+
+There are instructions on how to install the stable branch of Nginx [here](https://www.digitalocean.com/community/tutorials/how-to-install-the-latest-version-of-nginx-on-ubuntu-12-10).
+
+uWSGI you can install with `pip`.
+
+
+Getting Wooey to listen on Port 80
+-------------------
+
+You could at this point have Wooey listen to port 80 (assuming it's open) with the following command, run from the folder containing manage.py:
+
+::
+
+    python manage.py runserver 0.0.0.0:80
+
+But this leaves us widly insecure, because all our passwords will be transmitted over HTTP. I had problems working with non-standard ports on UFW so for the purposes of this tutorial, so I set-up my ports using this [iptables linode tutorial](https://www.linode.com/docs/security/firewalls/control-network-traffic-with-iptables).
+
+
+Getting Wooey to talk to Nginx & uWSGI
+---------------------
+
+We will basically follow the tutorial [here](http://uwsgi-docs.readthedocs.io/en/latest/tutorials/Django_and_nginx.html). If you installed Wooey in a virtualenv from the get go, then follow all those steps, if not you can ignore that set:
+
+Then we can follow the guide along with a couple changes:
+
+1. Basic test
+
+    * Because I was having trouble using non-standard ports, I replaced 8000 with 80 and ran the commands as root
+
+2. Test your Django project
+
+    * In order to get this step to work correctly I found I had to call uWSGI from the main folder where manage.py lives, in my case that was /home/user/ProjectName (I have no idea why, but otherwise it won't find the Wooey project correctly)
+
+3. Deploying static files
+
+    * So far I have ignored this altogether and no problems...      
+
+4. nginx and uWSGI and test.py
+
+    * Again only got this to work on port 80 again because of non-standard port problems
+
+
+Now if you want to run on a network socket, at this point you should be good to go. (Remember crucially this needs to be run from the same folder as manage.py).
+
+::
+
+    uwsgi --socket 127.0.0.1:8000 --wsgi-file ProjectName/wsgi.py --chmod-socket=666
+
+If you want to use a file socket, I then created an empty file in the Wooey project directory to be used as one (in this example named django.sock).
+
+::
+
+    uwsgi --socket ProjectName/django.sock --wsgi-file ProjectName/test.py --chmod-socket=666
+
+
+At this point we should now have Wooey running on Port 80 through Nginx.
+
+Forcing SSL with Nginx
+---------------------
+
+I have forced SSL with the following settings. (I think I might be running two SSL redirects, one on the Nginx side and one on the Django side which is never necessary because Nginx comes first, any clarification would be welcome, however for those following along:)
```

Comment:
Yes, you do not need the SSL on the wooey side because termination is occurring at the nginx layer.